app/utils/ApiUtils.js

```diff
@@ -5,8 +5,17 @@ import xhr from '../lib/xhr';
 
 const ApiUtils = {
 
-  requestAvailabilities(date) {
-    xhr.getJSON(`${API}/facility/the-fish/availability?date__gte=${date}`, (err, res) => {
+  requestFacilityInfo(facilityId) {
+    xhr.getJSON(`${API}/facility/${facilityId}`, (err, res) => {
+      if (err) {
+        return err;
+      }
+      ServerActionCreators.facilityInfoLoaded(res);
+    });
+  },
+
+  requestAvailabilities(facilityId, date) {
```

Comment:
Not sure about passing the facilityId as an argument or taking it from the state, but after thinking about it I think this makes it more flexible, since we are passing the two variable parts as arguments. bad thing about this: we need to have access to facilityId every time we call this function.